### ISSUE TITLE
ID3 iTunes Podcast Tags

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dhamaniasad/mutagen#141]
```
Hi,

I've edited the id3.py file to include the ID3 tags iTunes uses to display a 
podcast's description, feed and identifier. These are:
TDES(TextFrame): "Description" # Podcast description
TGID(TextFrame): "Identifier" # Podcast identifier
WFED(UrlFrame): "Feed" # Podcast feed

Adding them was a simple matter of adding them as classes with the other tags 
(TIT2, TIT3, etc).

I'm a little unsure as to how to contribute to the project in terms of a patch 
etc., but hopefully this should make it easy enough for someone else to add the 
tags themselves if they so wish. (And any guidance on how to produce a patch 
for this would be great).


```

Original issue reported on code.google.com by `dave.r...@gmail.com` on 12 Apr 2013 at 8:03

**Comment by GoogleCodeExporter** [comment]
```
https://github.com/judwhite/IdSharp/pull/1
https://bitbucket.org/nicfit/eyed3/commits/cc06d3ce1d8714bf3c8fff9e4e6388bb2df3b
c94

Do you have an example file (and can send it to me)?

To create a patch:

hg clone https://code.google.com/p/mutagen/
cd mutagen
# change stuff
./setup test # make sure all tests pass
hg diff > foobar.patch
```

Original comment by `reiter.christoph@gmail.com` on 15 Apr 2013 at 3:30
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
I've attempted to send you a file via email; I've attached it here also. The 
changes are on lines 1332, 1341 and 1393. (So I've not added the tags for 
ID3v2.2)

Those links are interesting. I didn't have to do anything but add these tags in 
as extra classes. I had already changed the files to being podcast types via 
iTunes, though I believe that's just another custom ID3 tag? I'm not sure why 
those other projects are attempting to make entirely new frames etc.

I'll attempt to make a patch this evening.

Sorry for the delay in sorting this out.
```

Original comment by `dave.r...@gmail.com` on 23 Apr 2013 at 7:56

Attachments:
- [id3.py](https://storage.googleapis.com/google-code-attachments/mutagen/issue-141/comment-2/id3.py)

**Comment by GoogleCodeExporter** [comment]
```
I meant a mp3 file that contains the mentioned frames.

Does iTunes provide an interface to edit them?
```

Original comment by `reiter.christoph@gmail.com` on 23 Apr 2013 at 8:30

**Comment by GoogleCodeExporter** [comment]
```
Ah! Sorry.

iTunes doesn't, strangely. It only displays the description and orders by the 
publication date (which you can't edit, but was in the mutagen id3.py file 
already). I initially used ID3 Editor to check/edit the tags until editing the 
id3.py file: http://www.pa-software.com/id3editor/

I've had to email you the podcast due to the 10mb attachment limit (the 
smallest one I have that I've edited via the script is 13mb).
```

Original comment by `dave.r...@gmail.com` on 23 Apr 2013 at 8:39

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 16095505d66e.
```

Original comment by `reiter.christoph@gmail.com` on 5 Sep 2013 at 8:39
- Changed state: **Fixed**
